apps/openssl.c

```diff
@@ -296,12 +296,28 @@ static void list_md_fn(const EVP_MD *m,
     }
 }
 
+static void list_missing_help(void)
+{
+    const FUNCTION *fp;
+    const OPTIONS *o;
+
+    for (fp = functions; fp->name != NULL; fp++) {
+        if ((o = fp->help) == NULL)
```

Comment:
You're right, every command should at least have a -help :)  Updated commit pushed.